packages/text-field/Input.js

```diff
@@ -121,6 +121,7 @@ export default class Input extends React.Component {
 
   render() {
     const {
+      inputType,
```

Comment:
@bmihelac do you mind changing this from `inputType` to `tag`? I saw JamesFriedman from RMWC is using this prop name, and I like it! 

Also using it in drawer, so it will be consistent with the rest of the components that we will build.